tests/profiles/helm/values-istio-demo-auth.yaml

```diff
@@ -1,7 +1,7 @@
 # This is used to generate minimal demo mode. It is included from demo and demo-auth values.
```

Comment:
should profiles not come from the installer repo?  If not, I think that is ok, but we should take an opinion on this in the environments WG meeting.

Reply:
There is no such thing as 'profiles' in installer repo - it's supposed to be a-la-carte, each component has its own isolated config.

We do generate kustomize profiles - but as an output.

Reply:
they are coming from istio/istio and stored here because they are manually translated.